src/js/calculate.js

```diff
@@ -0,0 +1,67 @@
+import {
+  LENGTH_LIMIT_OPERATOR,
+  MAX_NUMBER_LENGTH,
+  MAX_OPERATOR_LENGTH,
+  OPERATORS,
+  SPLIT_NUMBER_ARRAY_LENGTH,
+} from './constants.js';
+import checkNotNumber from './regex.js';
+
+const handleDecimalPointFloor = (countedNumber) => String(Math.floor(countedNumber));
+
+const handleCalculate = (beforeNumber, operator, afterNumber) => {
+  const { plus, minus, multiplication, divide } = OPERATORS;
+  switch (operator) {
+    case `${plus}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) + parseInt(afterNumber, 10));
+    case `${minus}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) - parseInt(afterNumber, 10));
+    case `${multiplication}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) * parseInt(afterNumber, 10));
+    case `${divide}`:
+      return handleDecimalPointFloor(parseInt(beforeNumber, 10) / parseInt(afterNumber, 10));
+    default:
+      break;
+  }
+};
+
+const handleClickEqual = (operatorText, input) => {
+  const findOperatorArray = input.match(checkNotNumber);
+  // NOTE : input에 연산자가 없거나, 연산자가 있지만 연산자 이후 바로 "="이 들어오는 경우 계산없이 받아온 input 그대로 return
+  if (!findOperatorArray || input[input.length - 1] === String(findOperatorArray)) {
+    return input;
+  }
```

Comment:
가능하면 주석보다는 코드로 충분히 표현하는 게 더 관리하기 좋습니다!
```suggestion
  const hasNoOperator = !findOperatorArray
  const hasNoRightValue = input[input.length - 1] === String(findOperatorArray)
  if (hasNoOperator || hasNoRightValue) {
    return input;
  }
```

Reply:
주석없이 변수명으로 주석의 역할을 대체할수있다는게 정말 인상깊네요!
앞으로 주석보다는 코드로, 변수명으로 표현하도록 노력하겠습니다 감사합니다 좋은 인사이트 얻었습니다!